plugins/nanospell/plugin.js

```diff
@@ -313,28 +327,33 @@
 
 			/* #2 setup layer */
 			/* #3 nanospell util layer */
-			var start = function () {
+			function start() {
 				editor.getCommand('nanospell').setState(CKEDITOR.TRISTATE_ON);
 				state = true;
-				var words = getWords(editor.document.$.body, maxRequest);
-				if (words.length == 0) {
-					render();
-				} else {
-					send(words);
-				}
-			};
-			var stop = function () {
+
+				startSpellCheckTimer(50);
```

Comment:
Can we create a constant to store that delay. Looks like it's used in one other place.